pkg/scyllaclient/client_rclone_agent_integration_test.go

```diff
@@ -58,7 +58,7 @@ func TestRcloneSkippingFilesAgentIntegration(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	id, err := client.RcloneCopyDir(ctx, testHost, remotePath(""), "data:tmp/copy")
+	id, err := client.RcloneCopyDir(ctx, testHost, remotePath(""), "data:tmp/copy", "")
```

Comment:
I don't see any integration test validating usage of "suffix" option.
Please add. 
It's not enough to test it just with the backup / restore.